.repo-metadata.json

```diff
@@ -11,7 +11,7 @@
   "repo": "googleapis/java-iam",
   "repo_short": "java-iam",
   "library_type": "OTHER",
-  "distribution_name": "com.google.api.grpc:proto-google-iam-v1",
+  "distribution_name": "com.google.cloud:google-iam-policy",
```

Comment:
Memo: OwlBot Postprocessor modifies versions.txt and other files based on repo-metadata.json.

https://github.com/googleapis/synthtool/blob/7197a001ffb6d8ce7b0b9b11c280f0c536c1033a/docker/owlbot/java/src/fix-poms.py#L279